meetings/2021-07-28.md

```diff
@@ -0,0 +1,136 @@
+# 2021-07-28 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+
+## Agenda
+
+* Minutes - https://github.com/solid/authorization-panel/pull/240
+* UC0 - https://github.com/solid/authorization-panel/pull/244
+* UC3 - https://github.com/solid/authorization-panel/pull/241
+* ACL on ACL - https://github.com/solid/authorization-panel/issues/189
+* Mode range: https://github.com/solid/authorization-panel/issues/187
+
+## Present
+
+* Elf Pavlik
+* Sarven Capadisli
+* Henry Story
+* Matthieu Bosquet
+
+
+## Minutes
+
+## UC0
+
+Sarven: What happens when you get a 403 on ACR?
+
+Matthieu: I don't think we have a good required credentials discovery mechanism. That's what we need defining.
+
+Sarven: You have foo and foo.acl. If I get a 404 on foo.acl, it tells me that foo potentially exists. It emphasises on the authorise. If you know that foo exists but don't have access to it, getting a 403 (or 404) is fine.
+
+Elf: Go to foo, get a header to the acl, how do you know?
+
+Sarven: Whether you know or not that foo.acl exists. It's fine, but if you don't know it exists, what should happen?
+
+Elf: Without knowing that foo exists, you don't get to know that the acl exists.
+
+Sarven: If you don't know that foo exists you should get a 403.
+
+Elf: How do you get a response you don't get a response.
```

Comment:
Maybe:
```suggestion
Elf: How would you get to a resource `foo` if you don't know it exists?
```

Reply:
The original is unclear enough that I'd like to get @elf-pavlik to weigh in, even though @matthieubosquet's suggestion seems likely.